app/views/admin/orders/_order.html.erb

```diff
@@ -1,6 +1,6 @@
 <td><%= "Order #{order.id}" %></td>
 <td><%= order.address.name %></td>
-<td><%= price_in_currency(order.payments.first.total_amount) %></td>
+<td><%= price_in_dollar(order.payments.first.total_amount/Currency.rate_on(order.currency, order.order_at.to_date)) %></td>
```

Comment:
let's use preferred currency wherever user is signed in

Reply:
done